policyengine_us/variables/gov/states/ma/dta/tcap/eaedc/income/ma_eaedc_total_earned_income.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class ma_eaedc_total_earned_income(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Massachusetts EAEDC total earned income"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MA
+
+    adds = ["ma_eaedc_earned_income"]
```

Comment:
make a formula and lets do the same as in disabled income just with `~disabled & ~dependent`